### PR TITLE
Policy Refresh H2 FY24

### PR DESCRIPTION
**Pull request by Springstone**
## Overview/Summary

This is the H2 FY24 Policy Refresh!

#### Azure Public

[Deploy To Azure](https://portal.azure.com/#view/Microsoft_Azure_CreateUIDef/CustomDeploymentBlade/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FEnterprise-Scale%2Fpolicy-refresh-q3fy24%2FeslzArm%2FeslzArm.json/uiFormDefinitionUri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FEnterprise-Scale%2Fpolicy-refresh-q3fy24%2FeslzArm%2Feslz-portal.json)

## As part of this Pull Request I have

- [x] Checked for duplicate [Pull Requests](https://github.com/Azure/Enterprise-Scale/pulls)
- [x] Associated it with relevant [issues](https://github.com/Azure/Enterprise-Scale/issues), for tracking and closure.
- [x] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/Enterprise-Scale/tree/main)
- [x] Performed testing and provided evidence.
- [x] Ensured [contribution guidance](https://github.com/Azure/Enterprise-Scale/wiki/ALZ-Contribution-Guide) is followed.
- [x] Updated relevant and associated documentation.
- [x] Updated the ["What's New?"](https://github.com/Azure/Enterprise-Scale/wiki/Whats-new) wiki page (located: `/docs/wiki/whats-new.md`)

### CODE REVIEW

File: docs/wiki/Whats-new.md [hunk 1]
```diff
@@ -43,6 +45,74 @@ This article will be updated as and when changes are made to the above and anyth
 
 Here's what's changed in Enterprise Scale/Azure Landing Zones:
 
+### 🆕 AMA Updates
```
Comment (jtracey93) [inline docs/wiki/Whats-new.md:48]: Same here for @arjenhuitema, do we also need to include a list of assignments that ALZ Portal/ARM/Bicep customers need to manually remove so they do not have duplicate assignments if they used the previous policies?

Can we just provide a bullet point list for the policies and assignment scopes they need to remove manually?

**Review by jtracey93 — CHANGES_REQUESTED** [review]
2 changes please @Springstone & @arjenhuitema to the docs and then I think we are good.

I have also tested an upgrade deployment and am happy 😄 so just the docs piece and we are good to merge and release 

Cc: @paulgrimley

File: docs/wiki/Whats-new.md [hunk 2]
```diff
@@ -43,6 +45,74 @@ This article will be updated as and when changes are made to the above and anyth
...
+    - [[Preview]: Enable ChangeTracking and Inventory for virtual machines](https://www.azadvertizer.net/azpolicyinitiativesadvertizer/92a36f05-ebc9-4bba-9128-b47ad2ea3354.html)
+    - [[Preview]: Enable ChangeTracking and Inventory for virtual machine scale sets](https://www.azadvertizer.net/azpolicyinitiativesadvertizer/c4a70814-96be-461c-889f-2b27429120dc.html)
+- Custom [Defender for SQL initiative](https://raw.githubusercontent.com/Azure/Enterprise-Scale/main/src/resources/Microsoft.Authorization/policySetDefinitions/Deploy-MDFC-DefenderSQL-AMA.json) has been deprecated and is replaced by [Configure SQL VMs and Arc-enabled SQL Servers to install Microsoft Defender for SQL and AMA with a user-defined LA workspace](https://www.azadvertizer.net/azpolicyinitiativesadvertizer/de01d381-bae9-4670-8870-786f89f49e26.html)
+- Custom [User Assigned Managed Identity policy](https://raw.githubusercontent.com/Azure/Enterprise-Scale/main/src/resources/Microsoft.Authorization/policyDefinitions/Deploy-UserAssignedManagedIdentity-VMInsights.json) has been deprecated. UAMI for AMA is now centrally deployed therefore this policy is no longer required.
+
+### 🔃 Policy Refresh H2 FY24
```
[the remark this answers is not in the document]
Comment (Springstone) [reply docs/wiki/Whats-new.md:74]: See PR #1655

**Comment by nekdima** [comment]
Hey Team, could you please update Excel (ALZ Policy Assignments v2) with new policies? 
https://github.com/Azure/Enterprise-Scale/wiki/ALZ-Policies

**Comment by Springstone** [comment]
> Hey Team, could you please update Excel (ALZ Policy Assignments v2) with new policies? https://github.com/Azure/Enterprise-Scale/wiki/ALZ-Policies

@nekdima please note the Excel only includes policies we assign by default. The new policies and initiatives are not enabled by default and require selecting scopes to apply at. We'll look at an additional Excel that includes all ALZ policies/initiatives regardless of assignment in a future update.

**Pull request merged**